src/api-legacy/v1/action.py

```diff
@@ -96,6 +96,37 @@ def getordering(self, orderby, direction):
         elif orderby == 'creation':
             _orderby = 'created_bho'
 
+        elif orderby == 'workgroup':
```

Comment:
Q: Wenn ich das richtig verstehe, ist dann eine Sortierung nach _Workgroup_ völlig willkürlich da hier, wie von dir beschrieben, nach der _Id_, resp. dem _FK_ soritert wird. Ich habe den Backend Code nicht angeschaut. Aber wäre es allenfalls möglich die Workgroup Tabelle zu joinen und dann nach dem Namen zu sortieren?

Reply:
Nach kurzer Analyse mit @MiraGeowerkstatt würde sich eine entsprechende Erweiterung nur mit verhältnismässig (zu) grossem Aufwand realisieren lassen. Das in diesem Bereich keine Tests vorhanden sind, laufen wir zudem Gefahr, ungewollt andere Bereiche zu tangieren. In gewissen Fällen macht eine Sortierung nach _Id_ auch durchaus Sinn, da darüber indirekt und implizit eine Reihenfolge abgebildet wird (bspw. Status). Ausserdem ist auf dem UI nicht klar ersichtlich ist, nach was genau sortiert wird. Wir belassen es so wie es ist.